src/jobs/build-system.yml

```diff
@@ -22,14 +19,14 @@ steps:
       version: $ELIXIR_VERSION
   - install-hex-rebar
   - install-nerves-bootstrap
-  - br2-primary-site
   - mix-deps-get
   - when:
       condition: << parameters.hex-validate >>
       steps:
         - run:
             name: Validate Hex package
             command: mix hex.build
+  - restore-nerves-downloads
```

Comment:
I think that we might need to restore the downloads before doing the hex-validate method above. Calling `mix hex.Build` will attempt to compile and without the restored cache, that will take a while right?

I believe this hex-validate step is mostly called before releasing

Reply:
Good call, I've updated this in 42c55e3.